ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -130,125 +115,75 @@ Install the dependencies for the four packages which you acquired from Git. For
 
-```
-# apt-get install libayatana-ido3-0.4-dev
-# apt-get install libayatana-indicator3-dev
-# apt-get install libcanberra-dev
-# apt-get install libgnome-desktop-3-dev
-# apt-get install libgtk-3-dev
-# apt-get install liblightdm-gobject-1-dev
-# apt-get install valac
-# apt-get install xvfb
-# apt-get install fonts-cantarell
-# apt-get install fonts-cabin
-# apt-get install gnome-settings-daemon
-```
+- **`# apt-get install libayatana-ido3-0.4-dev libayatana-indicator3-dev libcanberra-dev libgnome-desktop-3-dev libgtk-3-dev liblightdm-gobject-1-dev valac xvfb fonts-cantarell fonts-cabin
+gnome-settings-daemon`**
 
 Install libpam-x2go dependencies, use the same command `apt-get install` followed by the dependencies:
 
-```
-# apt-get install libgtest-dev
-# apt-get install libpam0g-dev
-# apt-get install libssh-dev 
-# apt-get install cdbs 
-# apt-get install socat
-# apt-get install pyhoca-cli
-```
+- **`# apt-get install libgtest-dev libpam0g-dev libssh-dev cdbs socat pyhoca-cli`**
  
 Install the lightdm-remote-session-x2go dependencies, use the same command `apt-get install` followed by the dependencies:
 
-```
-# apt-get install dh-apparmor
-```
+- **`#apt-get install dh-apparmor`**
  
 install remote-logon-service dependencies, use the same command `apt-get install` followed by the dependencies:
 
-```
-# apt-get install dbus-test-runner  
-# apt-get install libdbustest1-dev
-# apt-get install libgcrypt11-dev 
-# apt-get install libjson-glib-dev 
-# apt-get install libnm-glib-dev
-# apt-get install libsoup2.4-dev
-```
+- **`# apt-get install dbus-test-runner libdbustest1-dev libgcrypt11-dev libjson-glib-dev libnm-glib-dev
+libsoup2.4-dev`**
 
 ## Stage five
 
 Install the devscripts, using same command `apt-get install` followed by the dependence:
 
-```
-# apt-get install devscripts
-```
+- **`# apt-get install devscripts`**
 
 Now navigate to each repository folder that was created, in order to open the folders you will need to type `cd` in front of each folder, though to go back one folder you will need to type `cd ..`
 
 Type following:
 
-```
-cd arctica-greeter
-```
+- **`# cd arctica-greeter`**
 
 Now you are in the folder here you will need to run the command:
 
-```
-debuild –uc –us
-```
+- **`# debuild –uc –us`**
 
-exit the folder by typing `cd ..` and navigate to the other folders and do the same
+exit the folder and type:
 
-```
-cd libpam-x2go
-cd lightdm-remote-session-x2go
-cd remote-logon-service
-```
+- **`# cd libpam-x2go && debuild -uc -us && cd lightdm-remote-session-x2go && debuild -uc -us && cd remote-logon-service && debuild -uc -us`**
 
 Go back to the folder where the 4 repositories were created using the command `cd ..` and install the created packages:
 
-```
-sudo dpkg -i *.deb
-```
+- **`# sudo dpkg -i *.deb`**
 
 In same folder install thin-client-config-agent:
 
-```
-# apt-get install thin-client-config-agent
-```
+- **`# apt-get install thin-client-config-agent`**
```

Comment:
same here... (there are 11 or so other occurrences, please re-fix them all) (they used to be ok in the previous comment accepted the trailing "#" or "$" (which you now added, but with that you went back to the broken code quoting format of yours).